final-project/index.html

```diff
@@ -1,14 +1,574 @@
+      fetch(url)
+        .then(function (data) {
+          return data.json();
+        })
+        .then(response => {
+          buildResourceItems(response);
+          var carouselResources = new Carourel({klass: '.resources__container-item', prefix: 'resources', withArrows: true});
+          carouselResources.init();
+        })
+        .catch(function (error) {
+          console.log('error', error);
+        });
+
+      function buildResourceItems(data) {
+        for(let i = 0; i < data.articles.length; i ++) {
+          let dataArt = data.articles[i];
+
+          let baseElm = document.querySelector('.resources__container');
+
+          let itemElm = document.createElement('div');
+          itemElm.className = 'resources__container-item item';
+
+          let contentElm = document.createElement('div');
+          contentElm.className = 'resources__container-item-content';
+
+          let imgElm = document.createElement('div');
+          imgElm.className = 'resources__container-item-content-image';
+          imgElm.style.backgroundImage = `url('${dataArt.images.desktop}')`;
+
+          let cardElm = document.createElement('div');
+          cardElm.className = 'resources__container-item-content-card';
+
+          let cardContElm = document.createElement('div');
+          cardContElm.className = 'resources__container-item-content-card-container';
+
+          let cardContTitle = document.createElement('div');
+          cardContTitle.className = 'resources__container-item-content-card-container-title';
+          let titleTxt = document.createTextNode(dataArt.title);
+          cardContTitle.appendChild(titleTxt);
+
+          let cardContDesc = document.createElement('div');
+          cardContDesc.className = 'resources__container-item-content-card-container-description';
+          let descTxt = document.createTextNode(dataArt.description);
+          cardContDesc.appendChild(descTxt);
+
+          let cardContActions = document.createElement('div');
+          cardContActions.className = 'resources__container-item-content-card-container-actions';
+
+          let btn = document.createElement("button");
+          let btnTxt = document.createTextNode("Read now");
+          btn.appendChild(btnTxt);
+
+          let p = document.createElement('p');
+          let pTxt = document.createTextNode("Add to your bookmarks");
+          p.appendChild(pTxt);
+
+          cardContActions.appendChild(btn);
+          cardContActions.appendChild(p);
+
+          cardContElm.appendChild(cardContTitle);
+          cardContElm.appendChild(cardContDesc);
+          cardContElm.appendChild(cardContActions);
+
+          cardElm.appendChild(cardContElm);
+
+          contentElm.appendChild(imgElm);
+          contentElm.appendChild(cardElm);
+
+          itemElm.appendChild(contentElm);
+
+          baseElm.appendChild(itemElm);
+        }
+      }
+    }
```

Comment:
As a suggestion to keep this simpler, both the carousel and this JS code could have been in their own JS, and imported by `scripts/main.js` to be bundled by webpack